apps/hash-graph/lib/graph/src/ontology/entity_type.rs

```diff
@@ -296,6 +296,23 @@ pub enum EntityTypeQueryPath<'p> {
     /// # Ok::<(), serde_json::Error>(())
     /// ```
     ///
+    /// It's also possible to create a query path for the reversed direction:
```

Comment:
How does this interact with external types? I assume that because there isn't a central repository, they won't show up until loaded in explicitly?

(non-blocking, just clarification)

Reply:
Yes, you cannot know if a type externally referencing this type, so it's not possible to read that.